test/widgets/lightbox_test.dart

```diff
@@ -249,7 +250,90 @@ void main() {
           matching: find.textContaining(findRichText: true,
             eg.otherUser.fullName)));
       check(labelTextWidget.text.toPlainText())
-        .contains('Jul 23, 2024 23:12:24');
+        .contains('Jul 23, 2024 at 11:12 PM');
+
+      debugNetworkImageHttpClientProvider = null;
+    });
+
+    testWidgets('app bar shows localized timestamp: a few seconds ago', (tester) async {
+      prepareBoringImageHttpClient();
```

Comment:
Why is this needed? (And the corresponding `debugNetworkImageHttpClientProvider = null;`.)

Similarly in the other added tests.